kolibri/plugins/management/assets/src/state/store.js

```diff
@@ -8,9 +8,9 @@ const initialState = {
   pageState: {},
   facility: undefined,
   users: [], // this should be inside page state
+  roles: [],
```

Comment:
Yeah, we should move it to pagestate eventually but it's fine for now.

It's not clear to me from the variable name what `roles` is. Is this an array of all possible roles? If so, this can probably be changed to a hard-coded list in constants.js rather than a vuex state variable.